src/loki/status_action.cc

```diff
@@ -1,8 +1,43 @@
+#include <boost/property_tree/json_parser.hpp>
+#include <iostream>
+#include <sstream>
+
+#include "baldr/tilehierarchy.h"
 #include "loki/worker.h"
+#include "proto/status.pb.h"
 
 namespace valhalla {
 namespace loki {
-void loki_worker_t::status(Api&) const {
+void loki_worker_t::status(Api& request) const {
+  // get _some_ tile
+  static baldr::graph_tile_ptr tile = nullptr;
+  if (!tile) {
+    for (const auto& tile_id : reader->GetTileSet()) {
```

Comment:
This is the only thing I'm worried about in this PR. This operation is potentially slow when running in tile dir mode. I would say we could just cache the results but it's possible that the caller wants to know what the uncached results are when the router is configured in an on the fly mode where the tileset is mutable

Reply:
IMO it's fine for it to be a slow operation as long as it's cached (which it is right now), but I'm very unclear on what to do about that dynamic operation mode of Valhalla, I have little experience with the different operation modes, no idea how to set that up even..

Is there a way you'd prefer? 

Can we detect it's running in dynamic mode and simply skip most attributes (and the tile lookup) to basically only give back the full `config`?

Reply:
my point is that its not cached when you arent using a tar, it will go to the filesystem and check which files are there: https://github.com/valhalla/valhalla/blob/master/src/baldr/graphreader.cc#L820

to me it seems a bit mean to have an implementation detail like this dictate the way this method works unless, like i was saying above, that someone really wants to see how their tile set is changing since the last call to this. to me though that seems like a very odd use-case. these top level apis are for down stream consumers that dont need to know the details of what is going on underneath so i think we can safely forget this usecase.

maybe we should do something like `locate` where we make use of the `verbose` option and only provide all this extra info when the person askes for `verbose==true`. that way you can opt into a longer more taxing request? any thoughts about that?

Reply:
got ya, thanks for the explanation.

adding `verbose` param (default false I guess) to the endpoint is fine with me. I'll do it that way then. 

Anything else you'd like to have in terms of tests @kevinkreiser ?

Reply:
tileset specific information is now only returned if `verbose: true`